LAMPAPI/UpdateContact.php

```diff
@@ -18,26 +18,32 @@
 		$stmt->bind_param("sss", $id, $login, $password);
 		$stmt->execute(); */
 
-		if (isset($firstName))
+		if (!empty($firstName))
```

Comment:
This is the change that takes care of the possibility of there being an empty field in the Update Contacts page @cereusware